share/site/duckduckgo/styleguide.tx

```diff
@@ -289,13 +289,15 @@
 			<hr class="hr--mid" />
 			
 			<div class="t-mid">
-				<form class="frm--newsletter" action="https://flash.duckduckgo.com/e.js" method="post" name="y">
-					<input type="hidden" name="add" value="1" />
-					<input type="text" placeholder="Enter your email address" class="frm__input" name="from" />
+				<form class="frm--newsletter" action="https://duck.co/s/a" method="post" name="y">
```

Comment:
We could remove the form elements just as easily. Suspect it may have been a c/p job initially.